scripts/cat/history.py

```diff
@@ -343,74 +351,73 @@ def add_possible_history(cat, condition:str, death_text:str=None, scar_text:str=
                     death_text = f"m_c died from an injury or illness ({condition})."
             if not scar_text:
                 scar_text = f"m_c was scarred from an injury or illness ({condition})."
-            
+
             cat.history.possible_history[condition] = {
                 "death_text": death_text,
                 "scar_text": scar_text,
                 "other_cat": other_cat.ID if other_cat else None
             }
 
-
     @staticmethod
     def remove_possible_history(cat, condition):
         """
         use to remove possible death/scar histories
         :param cat: cat object
         :param condition: condition linked to the death/scar you're removing
-        :param scar: set True if removing scar
-        :param death: set True if removing death
+        # :param scar: set True if removing scar
+        # :param death: set True if removing death
         """
 
         History.check_load(cat)
 
         if condition in cat.history.possible_history:
             cat.history.possible_history.pop(condition)
-    
+
     @staticmethod
-    def add_death(cat, death_text, condition=None, other_cat=None, extra_text=None):
```

Comment:
extra_text does nothing. I removed it but it's being called by death_events to try and add murder_unrevealed_history. I think it should stay removed (and remove it from there too) - and there should be an investigation into whether murder_unrevealed_history is working as intended.

Reply:
Addressed in my moon event refactoring :+1: